### ISSUE TITLE
Frameskip

### ISSUE THREAD

**Issue by keanine** [Rinnegatamante/DaedalusX64-vitaGL#97]
A frameskip option for games that run at high framerates would be useful in these early days to make more games playable. An example game would be Smash 64!

**Comment by bielgiroto** [comment]
Yeah, in games that run at 60fps, it could be used as a trick to save some performance, but in games that run at 30, or 20, it would make it even worse. But still, i honestly prefer some frames being skipped, than needing to hear a glitchy sound and when the game has some lag, it should make it feel more like an actual console tough, you never see a game with glitchy sound on the consoles when the framerate drops, they just skip the frames and keep the sound good.

(Slowdowns will always be bad, but making them less painful increases to the overall experience)

**Comment by Rinnegatamante** [comment]
You can usee the Sync Video Rate feeature for a feature similar to frameskip but that doesn't have the stuttering feelings usually frameskip brings.

**Comment by keanine** [comment]
Is the Sync Video Rate feature in the 0.3 release? The closest I could find was the V-Sync option which just causes slowdown when the emulator can't reach 60fps

**Comment by Rinnegatamante** [comment]
No, nightlies.

**Comment by bielgiroto** [comment]
Well, this will be closed then? If the Sync Video Rate option actually works better than frameskipping, i don't see why keep this issue, unless frameskip has benefits that are actually worth getting.

I tought it could be a great feature, but now that you said Sync Video Rate works better, maybe that option could be improved then? In some other emulators, when frameskipping is utilized, it skips some real frames to keep the virtual ones good and don't glitch the audio. As far as i know, the Sync Video option doesn't help that much on keeping the sound good, and Sync Audio Option, at the same way, doesn't help that much on keeping the sound "good", but are those options finished, or they still need improvements? 

I'm just curious, maybe working on a optimized frameskip option can give better results than working on those 2 options, idk.

**Comment by Rinnegatamante** [comment]
Sync Video Rate is not applicable in every game however. Some games depending on their internal video rate may slowdown when enabling it. I'll leave this open for referencee but for now there's no plan to re-implement frameskip as it was before.
